src/julia_internal.h

```diff
@@ -12,6 +12,18 @@
 #define sleep(x) Sleep(1000*x)
 #endif
 
+#ifndef __has_feature
+#define __has_feature(x) 0
```

Comment:
I got this from the [Clang documentation on this extension](http://clang.llvm.org/docs/LanguageExtensions.html#feature-checking-macros)